e2e/global/cli_test.go

```diff
@@ -120,26 +124,87 @@ func TestPromptExitCode(t *testing.T) {
 				return icmd.Command("docker", "system", "prune")
 			},
 		},
-		{
-			name: "plugin install",
-			run: func(t *testing.T) icmd.Cmd {
-				t.Helper()
-				return icmd.Command("docker", "plugin", "install", "--disable", "vieux/sshfs")
-			},
-		},
-		{
-			name: "plugin upgrade",
-			run: func(t *testing.T) icmd.Cmd {
-				t.Helper()
-				icmd.RunCmd(
-					icmd.Command("docker", "plugin", "install", "--disable", "--grant-all-permissions", "vieux/sshfs"),
-				)
-				icmd.RunCmd(
-					icmd.Command("docker", "plugin", "disable", "vieux/sshfs"),
-				)
-				return icmd.Command("docker", "plugin", "upgrade", "vieux/sshfs:latest", "vieux/sshfs:next")
-			},
-		},
+		// {
+		// 	name: "plugin install",
+		// 	run: func(t *testing.T) icmd.Cmd {
+		// 		t.Helper()
+		// 		skip.If(t, versions.LessThan(environment.DaemonAPIVersion(t), "1.44"))
```

Comment:
This looks like some temporary commit; might be better to squash these (re-shape commits so that they can be reviewed individually)

Reply:
same here, I temporarily committed this to be able to share what I had since this part I had no clue how the plugin system worked. So I reached out on slack about it using the commit reference. I'll clean up PRs in the future with squashes